### DISCUSSION TITLE
Update contributors list in README

### DISCUSSION THREAD

**Discussion by fballiano** [OpenMage/magento-lts#1759] [General]
Hi everybody, I've run the all-contributors script against the v19 branch and found out that:

```
Missing contributors in .all-contributorsrc:
    daim2k5, fballiano, jouriy, digitalpianism, justinbeaty, ADDISON74, aredridel, drwilliams, hhirsch, kdckrs, sicet7, theroch, AlterWeb, Caprico85, davidwindell, drashmk, lamskoy, ferdiusa, himanshu007-creator, idziakjakub, JosephMaxwell, joshua-bn, KBortnick, elidrissidev, MrGekko, nikkuexe, psdevhh, PatrickCronin, petrsvamberg, rafaelstz, mageconsult, sunel, ktomk, ToonSpin, wexo-team, wilfriedwolf, akrzemianowski, andthink, eetzen, lemundo-team, mdlonline

Unknown contributors found in .all-contributorsrc:
loekvangool, samsteele, nintenic
```

I think there's no way to know if a person contributed just to code or also docs etc.. so, should we add those people with the "code" tag?

**Comment by Flyingmana** [comment]
better would be to run it on the v20 branch to crawl contributors as it includes more commits. But that does not seem to be the issue here.

But actually, I wonder why it did not find this commits?
* https://github.com/OpenMage/magento-lts/pull/895/commits/5d6d48b93d601ffbc9398ae5386a551e1266a366
* https://github.com/OpenMage/magento-lts/pull/1015/commits/bac5735eeb0ec6a07455f4f36ea7373ebea21fd5

not sure where it got nintenic from, the account does at least not exist anymore. Maybe it got deleted and thats why its not finding it anymore.

**Reply by fballiano** [reply]
I've run it now on the v20:

```
Missing contributors in .all-contributorsrc:
    digitalpianism, justinbeaty, ADDISON74, aredridel, drwilliams, hhirsch, kdckrs, sicet7, theroch, AlterWeb, Caprico85, davidwindell, drashmk, lamskoy, ferdiusa, himanshu007-creator, idziakjakub, JosephMaxwell, joshua-bn, KBortnick, elidrissidev, MrGekko, nikkuexe, psdevhh, PatrickCronin, petrsvamberg, rafaelstz, mageconsult, sunel, ktomk, ToonSpin, wexo-team, wilfriedwolf, akrzemianowski, andthink, eetzen, lemundo-team, mdlonline

Unknown contributors found in .all-contributorsrc:
loekvangool, samsteele, nintenic
```

although maybe the branch doesn't matter? maybe it rungs against the whole repo? no idea.

I wouldn't remove the "unknown contributors", I'd just add the other ones if that's ok

**Reply by Flyingmana** [reply]
its ok, especially as its seemingly not working correctly for them (as we found 2 commits mentioning them)
